rxfingerprint/src/main/java/com/mtramin/rxfingerprint/FingerprintObservable.java

```diff
@@ -115,6 +117,20 @@ public void onAuthenticationSucceeded(FingerprintManagerCompat.AuthenticationRes
 	}
 
 	/**
+	 * Required to catch errors from certain phones. See:
+	 * http://stackoverflow.com/questions/36043912/error-after-fingerprint-touched-on-samsung-phones-android-security-keystoreexce
+	 * @param e Exception caught
+	 * @param subscriber Subscriber for fingerprint result
+     */
+	void checkKey(Exception e, Subscriber<? extends FingerprintAuthenticationResult> subscriber, Action0 action0) {
```

Comment:
While that might be the case, and certainly an issue worth fixing it's not addressing the issue as described in #24. I'd therefore handle these cases in another issue/PR.